reactComponents/src/searchbar/tokenSearch/ResultDetail.tsx

```diff
@@ -0,0 +1,328 @@
+
+import React, { useContext, FC } from 'react';
+import styled from 'styled-components'
+import DefaultIcon from '../icons/default';
+import { Logo } from './Logo';
+import { firstAndLast } from './helpers/firstAndLast';
+import { intToWords } from './helpers/intToWords';
+import TokenSearchContext from '../Context/TokenSearch';
+
+import DownIcon from '../icons/down';
+import UpIcon from '../icons/up';
+const imageSize = 26;
+
+const StyledDetailList = styled.div`  
+  ${({styles}) => `
+    display: ${ styles?.container?.display || "grid" };
```

Comment:
Is there a linter in the terminal-library project? I see inconsistent code style. On this line, for example, there are spaces within the interpolation braces whereas there are other cases where there are not.

Reply:
Please read the readme, and if you have good idea, please let me know.